ui/src/components/NodePagePodsTab.js

```diff
@@ -156,11 +57,34 @@ const NodePagePodsTab = (props) => {
       {
         Header: 'Name',
         accessor: 'name',
+        cellStyle: {
+          flexGrow: '4',
+          width: '100%',
+          maxWidth: '350px',
+          minWidth: '200px',
```

Comment:
We should use `rem` as the unit instead of `px`. 
16px = 1rem
Please make sure to check the display on the small screen as well ;)
![image](https://user-images.githubusercontent.com/18453133/149797091-24f0ade2-6d70-4eaa-9590-23d428b41b92.png)